### PR TITLE
Set default of "table_manipulation" as "replace" in API endpoint when users enter None and updated tests

### PR DESCRIPTION
**Pull request by linglp**
I noticed that the `table_manipulation` is using `replace` as the default option. ~~If that's the case, table_manipulation parameter should be set as "required" so that users can't set an empty string or None to `table_manipulation` parameter on swagger UI~~.  To match with the downstream submit function, I set "table_manipulation" as an optional parameter, and if users don't set anything, we would pass "replace" by default. I also updated the description of the parameter for clarify. 

Context:  I discovered this issue when running tests for #1101

### CODE REVIEW

File: api/openapi/api.yaml [hunk 1]
```diff
@@ -303,6 +303,7 @@ paths:
           schema:
             type: string
             enum: ["replace", "upsert"]
+          required: true
```
Comment (linglp) [inline api/openapi/api.yaml:306]: If not setting this as "required", users could pass in None for "table_manipulation" which could end up breaking the downstream function.

**Review by mialy-defelice — CHANGES_REQUESTED** [review]
Is the change to generator related to a different issue? Even though its so tiny maybe we should move it to a different PR for clarity?

**Comment by linglp** [comment]
> Is the change to generator related to a different issue? Even though its so tiny maybe we should move it to a different PR for clarity?

Thanks for checking Mialy! The change to the generator is related to version updates of pandas. I was running tests for #1101, #1092 and realized that the pandas version has changed in the `develop` branch at one point (and I was still using an older version of pandas locally). After #1092 gets merged and `develop-synclient-branch` pulled from the new develop branch, this line won't show as a "change"

**Review by mialy-defelice — APPROVED** [review]
Looks great!

**Review by GiaJordan — APPROVED** [review]
Thanks @linglp! Just confirming, the necessary merge order is #1092, this one, and then #1101?

**Comment by GiaJordan** [comment]
Also, @linglp from the title it seems like you're making `--table_manipulation` required, but from the code it seems like you're making it optional. Should we update the title?

**Pull request merged**